src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_DeconstructionAssignmentOperator.cs

```diff
@@ -54,6 +54,7 @@ public override BoundNode VisitDeconstructionAssignmentOperator(BoundDeconstruct
                 stores.Add(assignment);
             }
 
+            stores.Add(new BoundVoid(node.Syntax, node.Type));
             var result = _factory.Sequence(temps.ToImmutable(), stores.ToArray());
```

Comment:
Rather than adding BoundVoid node to the stores array, it would be more efficient to call a different overload of Sequence and pass the node separately.